src/Converter.cc

```diff
@@ -29,12 +29,18 @@
 #include "sdf/Types.hh"
 
 #include "Converter.hh"
-
-// This include file is generated at configure time.
-#include "sdf/EmbeddedSdf.hh"
+#include "EmbeddedSdf.hh"
 
 using namespace sdf;
 
+namespace {
+bool EndsWith(const std::string& a, const std::string& b)
+{
+  return (a.size() >= b.size()) &&
+      (a.compare(a.size() - b.size(), b.size(), b) == 0);
+}
```

Comment:
Style: function parameters should start with underscore 
```suggestion
bool EndsWith(const std::string &_a, const std::string &_b)
{
  return (_a.size() >= _b.size()) &&
          (_a.compare(_a.size() - _b.size(), _b.size(), _b) == 0);
}

```